### PR TITLE
Added new pgm PG2 to Release branch

### PR DESCRIPTION
**Pull request by PesalaU123**

### CODE REVIEW

File: PG2.NSP [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+* >Natural Source Header 000000
+* :Mode S
+* :CP
+* <Natural Source Header
+/** New Program PGM1.
+/**
+/** :author Administrator
+/* TODO Enter your code here
+write 'WELCOME TO GIT HUB TRAINING'
+write 'day2'
```
Comment (vsb9962011485) [inline PG2.NSP:11]: Adding comment to include new write statement for date

**Review by vsb9962011485 — COMMENTED** [review]
Please fix code review comments

File: PG2.NSP (hunk 1 above)
Comment (PesalaU123) [reply PG2.NSP:11]: Fixed

**Review by vsb9962011485 — APPROVED** [review]
Approved

**Pull request merged**